src/pipelines/floodscan_pipeline.py

```diff
@@ -373,6 +376,35 @@ def combine_bands(self, sfed, mfed, date):
                     f"Failed when combining sfed and mfed geotiffs. {err}"
                 )
 
+    def _retrieve_datarray_for_date(self, date, sfed_filename, sfed_local_file_path):
+        if self.mode == "local":
+            if sfed_local_file_path.exists():
```

Comment:
Have tested running locally and this causes a downstream error because there's no `else` condition here. So the `da_in["date"] = date` below fails if the local path doesn't exist because `da_in` isn't defined here. 

I think we either want more informative logging to tell the user what they need to do to get this to pass, or to automatically download the data that is needed.